src/migrations/1697479763911_make-all-words-discoverable.ts

```diff
@@ -0,0 +1,48 @@
+import { Type } from "decentraland-gatsby/dist/entities/Database/types"
+import { ColumnDefinitions, MigrationBuilder } from "node-pg-migrate"
+
+import PlaceModel from "../entities/Place/model"
+
+export const shorthands: ColumnDefinitions = {
+  hidden: {
+    type: Type.Boolean,
+    default: false,
+    notNull: true,
+  },
+}
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.dropIndex(PlaceModel.tableName, ["hidden", "world_name"])
+  pgm.dropIndex(PlaceModel.tableName, ["hidden", "base_position"])
+  pgm.dropIndex(PlaceModel.tableName, ["hidden", "updated_at"])
+  pgm.dropIndex(PlaceModel.tableName, ["hidden", "like_rate"])
+
+  pgm.dropColumn(PlaceModel.tableName, shorthands)
+
+  pgm.createIndex(PlaceModel.tableName, ["world_name"])
+  pgm.createIndex(PlaceModel.tableName, ["base_position"])
+  pgm.createIndex(PlaceModel.tableName, ["updated_at"])
+  pgm.createIndex(PlaceModel.tableName, ["like_rate"])
+}
```

Comment:
Here you'll need to drop the same index created before and then create the new index with the same where but without the hidden, in order to make a better perform.

```suggestion
export async function up(pgm: MigrationBuilder): Promise<void> {
  pgm.dropIndex(PlaceModel.tableName, ["hidden", "world_name"])
  pgm.dropIndex(PlaceModel.tableName, ["hidden", "base_position"])
  pgm.dropIndex(PlaceModel.tableName, ["updated_at"])
  pgm.dropIndex(PlaceModel.tableName, ["like_rate"])

  pgm.dropColumn(PlaceModel.tableName, shorthands)

  pgm.createIndex(PlaceModel.tableName, ["world_name"], {
    where: "disabled is false and world is true",
  })
  pgm.createIndex(PlaceModel.tableName, ["base_position"], {
    where: "disabled is false and world is false",
  })
  pgm.createIndex(PlaceModel.tableName, ["updated_at"], {
    where: "disabled is false and world is false",
  })
  pgm.createIndex(PlaceModel.tableName, ["like_rate"], {
    where: "disabled is false and world is false",
  })
}
```